Snowflake.Data.Tests/IntegrationTests/MaxLobSizeIT.cs

```diff
@@ -82,7 +81,7 @@ public void SetUp()
                 using (var command = conn.CreateCommand())
                 {
                     // Create temp table
-                    var columnNamesWithTypes = $"{s_colName[0]} VARCHAR, {s_colName[1]} VARCHAR, {s_colName[2]} INT";
+                    var columnNamesWithTypes = $"{s_colName[0]} VARCHAR({MaxLobSize}), {s_colName[1]} VARCHAR({MaxLobSize}), {s_colName[2]} INT";
```

Comment:
why do you add size of the varchar? Is it needed for a reason?

Reply:
Yes, without it the size would be default (which differs for different max lob sizes and makes the test flaky)